website/sidebars.json

```diff
@@ -23,11 +23,10 @@
     "References": [
       "debugging-reference",
       "performance-guide",
-      "cli-reference",
```

Comment:
Removed this, since it isn't located here. This was causing the `next` button on the bottom of the `performance-guide` page to 404 and just say `next` rather than the actual name of the next page.